phantazm-mob/src/main/java/com/github/phantazmnetwork/mob/config/target/SelfSelectorConfigProcessor.java

```diff
@@ -0,0 +1,19 @@
+package com.github.phantazmnetwork.mob.config.target;
+
+import com.github.phantazmnetwork.mob.target.SelfSelector;
+import com.github.steanky.ethylene.core.ConfigElement;
+import com.github.steanky.ethylene.core.collection.LinkedConfigNode;
+import com.github.steanky.ethylene.core.processor.ConfigProcessor;
+import org.jetbrains.annotations.NotNull;
+
+public class SelfSelectorConfigProcessor implements ConfigProcessor<SelfSelector> {
+    @Override
+    public @NotNull SelfSelector dataFromElement(@NotNull ConfigElement element) {
+        return new SelfSelector();
+    }
+
+    @Override
+    public @NotNull ConfigElement elementFromData(@NotNull SelfSelector selfSelector) {
+        return new LinkedConfigNode();
```

Comment:
initial capacity of 0?

Reply:
shouldn't really matter if we don't modify it right?

Reply:
still takes up memory